### ISSUE TITLE
Sonos Bridge appearing as accessory

### ISSUE THREAD

**Issue by Garywoo** [ebaauw/homebridge-zp#97]
I've just noticed a Bridge accessory in my HomeKit accessory list. I don't remember this appearing as a HomeKit accessory before. Can this be optionally exposed, as it does not have any controls associated with it?

Running homebridge-zp v1.0.8

**Comment by ebaauw** [comment]
What's you config.json ?  If you set `leds`, the bridge should show as an accessory, with a LED service to switch the LED on or off and a Child Lock service to disable the physical buttons.  Otherwise, it shouldn't be exposed.

**Comment by Garywoo** [comment]
I don't have `leds` set.

Here's the ZP excerpt from config.json:
```
{
    "platform": "ZP",
    "service": "speaker",
    "brightness": true,
    "speakers": true,
    "searchTimeout": 15
},
```

**Comment by ebaauw** [comment]
Hm, I fear I broke this when fixing #96.

**Comment by ebaauw** [comment]
Should be fixed in v1.0.9.

**Comment by Garywoo** [comment]
Yep, looks okay now. Thanks!
